elastic/insta_user-elastic_filter/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.13 as builder
+WORKDIR /app
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -installsuffix cgo -o exe elastic/insta_user-elastic_filter/main.go
+FROM alpine
```

Comment:
FYI:
Adding a newline before the from makes it easier to see that this is a multistage build